gradle-baseline-java/src/main/resources/greclipse.properties

```diff
@@ -0,0 +1,3 @@
+org.eclipse.jdt.core.formatter.tabulation.char=space
+org.eclipse.jdt.core.formatter.indent_empty_lines=false
+groovy.formatter.remove.unnecessary.semicolons=true
```

Comment:
I did a trial run with this config on devx/template-witchcraft and it makes _zero_ changes!! I didn't quite believe it, so I indented stuff like crazy and it restored it :)